src/radical/entk/appman/appmanager.py

```diff
@@ -111,11 +97,8 @@ def __init__(self,
         self._prof.prof('amgr_creat', uid=self._uid)
```

Comment:
a little offtop, but just to make reporting nicer
L96 -> `self._report.info('Creating AppManager\n')`
L123 -> 
```
        self._report.info('AppManager initialized')
        self._report.ok('>>ok\n')
```

Reply:
I am not exactly sure what you mean - but I am all for improving the reporter output.  Would you mind pushing a patch?

Reply:
that is a small update for not to have stick together two messages (`Creating AppManagerSetting up ZMQ queues`)
and to have something like this
```
Creating AppManager
Setting up ZMQ queues                                                         ok
AppManager initialized                                                        ok
Validating and assigning resource manager                                     ok
Setting up ZMQ queues                                                        n/a

```

Reply:
Ah, of course :-)

Reply:
Done.

Reply:
tried it again,  and there is a line with lonely `ok`
```
EnTK session: re.session.b7da4236-9926-11ed-b1d0-0242c0a86005
Creating AppManager
Setting up ZMQ queues                                                         ok
                                                                              ok
```

let's add `self._report.info('AppManager initialized')` before [L130](https://github.com/radical-cybertools/radical.entk/blob/b877c876539b3dfa0fd4199cd95603114a20b013/src/radical/entk/appman/appmanager.py#L130)?

Reply:
Sure, done.